src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs

```diff
@@ -73,6 +76,13 @@ public override Task ShutdownAsync(CancellationToken cancellationToken)
 
         private Task SendSpansAsync(IEnumerable<SpanData> spans)
         {
+            if (this.tracer == null)
+            {
+                this.tracer = TracerFactoryBase.Default.GetTracer("OpenTelemetry");
+            }
+
+            using var scope = this.tracer.StartActiveSpan("OpenTelemetry.Zipkin.SendSpans", SpanKind.Terminal, out _);
```

Comment:
The issue you described about application insights telemetryclient doing its own  bacthing and making the http.send at some future point from different thread - this issue is not specific to application insights right? Any library can face this?

Reply:
@cijothomas Correct. It becomes the responsibility of the exporter lib author to properly flag the span as terminal (or set the CollectionMode=OFF) when it goes to transmit data. It's not ideal.

Reply:
mm. Got it.